BILL_OF_MATERIALS.md

```diff
@@ -0,0 +1,43 @@
+# Bill Of Materials (BOM)
+
+
+## Alexander Koch arms
+
+### Follower and Leader arms
+
+TODO
+
+### Leader arm only
+
+| Part | Amount | Cost/p US | Buy US | Cost/p EU | Buy EU | Cost/p UK | Buy UK |
```

Comment:
Nit: For `Cost/p US` is this cost per part? If so, it is a bit confusing because some of these items aren't individual parts, they come in, for example, packs of 4. I'm think maybe Unit Cost(US) or just Cost(US) would be better.